imsv-docs-astro/astro.config.mjs

```diff
@@ -33,7 +33,7 @@ export default defineConfig({
         {
           label: 'Guides',
           items: [
-            { label: 'Authentication', link: 'guides/authentication' } ,
+            { label: 'Authentication', autogenerate: { directory: 'guides/authentication' } },
             { label: 'HTTP Status Codes', link: 'guides/http-status-codes' } ,
             { label: 'Supported Regions', link: 'guides/regions' } ,
```

Comment:
I think these two unsectioned guides should be moved to the top of the Guides navigation

Reply:
I deliberately left it like this so we could have a conversation about what to do with the final two dangling items.